src/atoms/TextArea/__tests__/TextArea.web.test.js

```diff
@@ -0,0 +1,331 @@
+        <TextArea label={SAMPLE_LABEL} id={SAMPLE_ID} labelPosition="top" />,
+      );
+      const textArea = queryByLabelText(SAMPLE_LABEL);
+      expect(textArea).not.toBeNull();
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders TextArea with labelPosition on top(default) and variant filled', () => {
+      const { queryByLabelText, container } = renderWithTheme(
+        <TextArea label={SAMPLE_LABEL} id={SAMPLE_ID} labelPosition="top" variant="filled" />,
+      );
+      const textArea = queryByLabelText(SAMPLE_LABEL);
+      expect(textArea).not.toBeNull();
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders TextArea with labelPosition on left and variant filled', () => {
+      const { queryByLabelText, container } = renderWithTheme(
+        <TextArea label={SAMPLE_LABEL} id={SAMPLE_ID} labelPosition="left" variant="filled" />,
+      );
+      const textArea = queryByLabelText(SAMPLE_LABEL);
+      expect(textArea).not.toBeNull();
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('placeholder', () => {
+    it('should have default placeholder(empty) if placeholder is not passed as prop', () => {
+      const defaultPlaceholder = '';
+      const { getByPlaceholderText } = renderWithTheme(<TextArea />);
+      const textArea = getByPlaceholderText(defaultPlaceholder);
+      expect(textArea.placeholder).toEqual(defaultPlaceholder);
+    });
+
+    it('should update placeholder if placeholder is passed as prop', () => {
+      const placeholder = 'Example Placeholder';
+      const { getByPlaceholderText } = renderWithTheme(<TextArea placeholder={placeholder} />);
+      const textArea = getByPlaceholderText(placeholder);
+      expect(textArea.placeholder).toEqual(placeholder);
+    });
+  });
+
+  describe('width', () => {
+    it('renders TextArea with medium(default) width', () => {
+      const { container } = renderWithTheme(<TextArea width="medium" />);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders TextArea with small width', () => {
+      const { container } = renderWithTheme(<TextArea width="small" />);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders TextArea with auto width', () => {
+      const { container } = renderWithTheme(<TextArea width="auto" />);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('focus', () => {
+    it('should have focus when TextArea is focused', () => {
+      const { getByLabelText } = renderWithTheme(<TextArea label={SAMPLE_LABEL} id={SAMPLE_ID} />);
+      const textArea = getByLabelText(SAMPLE_LABEL);
+      textArea.focus();
+      expect(textArea).toHaveFocus();
+    });
+
+    it('should not have focus after TextArea blur', () => {
+      const { getByLabelText } = renderWithTheme(<TextArea label={SAMPLE_LABEL} id={SAMPLE_ID} />);
+      const textArea = getByLabelText(SAMPLE_LABEL);
+
+      // should have focus initially
```

Comment:
The comment looks slightly misleading. Focus is not initially. We are setting it. I think comment is not needed. Code is pretty self explanatory.